Week04/lesson08/homework/temp_converter_2/js/index.js

```diff
@@ -0,0 +1,36 @@
+function convertCToF() {
```

Comment:
That said, very cool that you got it converting upon typing in the input, good googling! 👍